docs/docs/guides/share/previews.md

```diff
@@ -0,0 +1,33 @@
+---
+title: Previews
+description: Learn how to generate and share previews of your apps with anyone.
+---
+
+# Previews
+
+
+When building an app, you may want to share it with others to get feedback.
+Traditionally, this is something that teams do by building, signing, and pushing their apps to platforms like Apple's [Testflight](https://developer.apple.com/testflight/).
+However, this process can be cumbersome and slow, especially when you're just looking for quick feedback from a colleague or friend.
+
+To make this process more streamlined, Tuist provides a way to generate and share previews of your apps with anyone.
+
+> [!IMPORTANT] ONLY SIMULATOR ARCHITECTURES ARE SUPPORTED
+> We only support simulator architectures for previews at the moment.
```

Comment:
```suggestion
> [!IMPORTANT] ONLY SIMULATOR ARCHITECTURES ARE CURRENTLY SUPPORTED
> We only support simulator architectures for previews at the moment. Support for devices will come in the future.
```

I'd be more explicit about the fact that support for devices is definitely on our roadmap.